### DISCUSSION TITLE
API for creating accounts

### DISCUSSION THREAD

**Discussion by mplayer143** [dani-garcia/vaultwarden#1382] [Q&A]
Hello everybody,

I don't really know if my request can be consider as an issue, but I want to know if someone can help me with this subject. 
I want to know if someone has ever tried to create an account by using the API called "register". 
I found that the API needs differents values : 
- keys
- publicKey
- encryptedPrivateKey
- masterPasswordHash
![Issue_Git_Bitwarden](https://user-images.githubusercontent.com/58776232/107630481-e08f9380-6c63-11eb-9d6a-8a7aa1434441.JPG)

I managed to rebuild the masterPasswordHash as the algo is known but for the others ... I don't know how they are given ... 

Thanks in advance for you replies :)

**Comment by xiki808** [comment]
Hey! Did you manage to find a way? I also have the same issue. 

How did you hash the Password?
How do you generate the Key?
How do you generate publicKey and encryptedPrivateKey?
